README.md

```diff
@@ -157,12 +157,6 @@ $ grunt bump:prerelease
 >> Tagged as "v1.0.0-1"
 >> Pushed to origin
 
-$ grunt bump:patch
```

Comment:
can you add the comments (in a separate PR) and unremove those lines in this PR?